### ISSUE TITLE
Bug: Issue with Folder and Disk Titles in the Preview Pane

### ISSUE THREAD

**Issue by Lukiluc29** [files-community/Files#12760]
### Description

I have noticed an issue regarding folder and disk titles in the preview pane. When navigating through folders such as "Downloads," for example, with the preview pane open, the folder title remains in English instead of being translated into French (since I am using Files in French). This occurs for all default Files folders, which means "Downloads," "Documents," "Videos," "Pictures," "Music," and "Desktop." Additionally, the disk titles are not displayed at all.

### Steps To Reproduce

1. Open Files
2. Go in the folder Downloads
3. Open the preview pane

### Requirements

Fix bugs or inconsistencies that prevent display of disc titles as well as incorrect translation of folders in the preview pane.

### Files Version

Version 2.5.10.0 / dev

### Windows Version

22H2 22621.1848

### Log File

![Capture d'écran 2023-06-27 110310](https://github.com/files-community/Files/assets/128183422/33094f73-5451-4474-a293-88338a6fab95)
![Capture d'écran 2023-06-27 110337](https://github.com/files-community/Files/assets/128183422/630143a1-7275-49d5-9f13-06cbf2e6ba0b)

**Comment by Josh65-2201** [comment]
Thanks for the report, This is international as the preview pane uses the folders name which are in English according to the path bar.

**Comment by Lukiluc29** [comment]
> Thanks for the report, This is international as the preview pane uses the folders name which are in English according to the path bar.

Yes, I noticed that this issue affects all other languages. Do you think there is still a solution to this?

**Comment by Josh65-2201** [comment]
Not sure if @yaira2 want to change it but I don't see this as an issue as it is the folder name

**Comment by Lukiluc29** [comment]
> Not sure if @yaira2 want to change it but I don't see this as an issue as it is the folder name

Yes, I understand, but it is somewhat inconvenient since Files is fully translated into a language other than English, and leaving words in English can be a bit bothersome. Furthermore, it would be beneficial to include the names of the disks in the preview pane, as currently it only displays the disk image without any accompanying text.

**Comment by yaira2** [comment]
What does the sidebar and tab show for these folders?

**Comment by Lukiluc29** [comment]
As I mentioned, I have Files set to French. Instead of displaying "Bureau" for the Desktop folder, it shows "Desktop" in English. Additionally, for the disks, no label is shown. For example, instead of displaying "Disque local (C:)" in the preview pane, there is no label at all.
![Capture d'écran 2023-06-27 172229](https://github.com/files-community/Files/assets/128183422/69c76305-82f4-454b-84d9-c25d0749c9a9)
![Capture d'écran 2023-06-27 172307](https://github.com/files-community/Files/assets/128183422/ac86400f-cfb4-4abc-a79b-544791118eb6)

**Comment by yaira2** [comment]
@Josh65-2201 if the folder is already localized in the sidebar, I think it's fair to do the same for the details pane.
